docs/tutorial/Create-the-bundle.md

```diff
@@ -1,75 +1,63 @@
-1.  <span>[Developer](index)</span>
-2.  <span>[Creating a Tweet Field Type](Creating-a-Tweet-Field-Type)</span>
-3.  <span>[Build the bundle](Build-the-bundle)</span>
+# Create the bundle
 
-<span id="title-text"> Developer : Create the bundle </span>
-============================================================
```

Comment:
I'd rather we stuck to one standard of headings, either atx or setext, and I think atx (the one with #) is better, because it covers the whole range from h1 to h6